src/client/dfuse/dfuse_core.c

```diff
@@ -78,11 +78,11 @@ dfuse_progress_thread(void *arg)
 static int
 dfuse_parse_time(char *buff, size_t len, unsigned int *_out)
 {
-	int		matched;
-	unsigned int	out = 0;
-	int		count0 = 0;
-	int		count1 = 0;
-	char		c = '\0';
+	int          matched;
```

Comment:
Add ` * (C) Copyright 2025 Google LLC` after Intel lines.  New githook will do this for you

Reply:
I think if you do an amend commit, it will do it if you have githook installed

Reply:
Done